README.md

```diff
@@ -25,3 +25,6 @@ npm -i -S @power-elements/lazy-image
 ```html
 <lazy-image src="/image.jpg" alt="Lazy Image"></lazy-image>
 ```
+
+### 'IntersectionObserver' in window
```

Comment:
LGTM. Do you have patience for a small nitpick? Would you please change the title of the new section in the README from `### 'IntersectionObserver' in window` to `### Browser Support`?